.github/workflows/deploy-solution.yml

```diff
@@ -0,0 +1,73 @@
+name: Deploy the solution
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push event but only for the "solution" branch
+  push:
+    branches: [ "solution" ]
```

Comment:
can we temporary trigger the work when pushing on the branch of this PR to test that the deployment can work?
This also means changing the configuration temporary and also fix it to at the end only authorize publish from the solution branch.

![image](https://user-images.githubusercontent.com/27200110/195860674-418e82fe-e687-4c91-bf5b-90f4152483bd.png)

Reply:
![image](https://user-images.githubusercontent.com/4921914/195865644-3145e661-719d-491a-b874-9bf991a00a99.png)

The [page](https://process-analytics.github.io/bpmn-visualization-tutorial-getting-started/) is empty 😭